src/palette/elements/Header/ArtsyLogoHeader.tsx

```diff
@@ -0,0 +1,29 @@
+import { ArtsyLogoIcon, Box, Flex, Spacer } from "palette"
+import React from "react"
+import { StyleSheet } from "react-native"
+
+interface Props {
+  shadow?: boolean
+}
+export const ArtsyLogoHeader: React.FC<Props> = ({ shadow = false }) => (
+  <>
+    <Box mt={2} mb={1} style={shadow && styles.boxShadowStyle}>
```

Comment:
I think giving `undefined` as a style here is not a good idea, maybe you can do
```suggestion
    <Box mt={2} mb={1} style={shadow ? styles.boxShadowStyle} : {}>
```